gdk4/Gir.toml

```diff
@@ -181,9 +181,8 @@ generate_builder = true
     name = "set"
     manual = true # use ToValue
     [[object.function]]
-    name = "read_async"
-    manual = true # array
-
+    pattern = "(read|read_texture|read_text|read_value|store)_async"
+    manual = true # don't enforce Send
```

Comment:
`concurrency = "none"` or not?

Reply:
Can only be set at object configuration level. also I believe the Send requirement comes from the _async_future variant somehow.

Reply:
Oh yes, can only be set global, per-object or for signals. Not for functions. That seems like something worth changing.

> also I believe the Send requirement comes from the _async_future variant somehow.

In what sense?

Reply:
> also I believe the Send requirement comes from the _async_future variant somehow.
Because the `_async` variant callback is called in the `async_future` one and there's no way to enforce `Send` there no?

Reply:
> Oh yes, can only be set global, per-object or for signals. Not for functions. That seems like something worth changing.

Opened an issue to handle that on gir side https://github.com/gtk-rs/gir/issues/1078

Reply:
> Because the _async variant callback is called in the async_future one and there's no way to enforce Send there no?

Why would the async_future variant need `Send` though?

Reply:
> Why would the async_future variant need Send though?

Because the future might be spawned on a different thread? I don't know. If it's not needed, gir should probably not automatically add the `Send` condition here then.

Reply:
I think that would panic already anyway? Can you make a small example that breaks?